packages/asset-uploader/package.json

```diff
@@ -40,14 +40,15 @@
     ".": {
       "source": "./src/index.ts",
       "import": "./lib/index.js",
-      "require": "./lib/cjs/index.js"
+      "require": "./lib/cjs/index.js",
+      "types": "./lib/types/index.d.ts"
```

Comment:
This should be moved right after the `source` condition everywhere. This currently works OK but relies on a TS bug (known as "🐛 Used fallback condition", you can learn more about it [here](https://arethetypeswrong.github.io/?p=%40webstudio-is%2Fasset-uploader%400.74.0)). 

`package.json#exports` are order-sensitive (even though they are defined using an object) and that rule should include `types` condition as well.